Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -155,36 +156,6 @@ static int CompareGameListItems(const GameListItem* iso1, const GameListItem* is
   return 0;
 }
 
-static std::vector<std::string> GetFileSearchExtensions()
```

Comment:
Why isn't this logic being used anymore? If you have lots of DOL and ELF files that you don't want to show in the game list, not scanning files that have .dol or .elf filename extensions improves the performance.